Tools/LauncherTestTools/run_launcher_tests_local_validation.py

```diff
@@ -36,26 +36,6 @@ def check_autoexe_cfg(dev_root_folder, project):
         if line.lower().startswith('map '):
             report_error_and_exit("Map command '{}' detected in {}".format(line.strip(), autoexec_cfg_path))
 
-def check_gems_enabled(dev_root_folder, project):
```

Comment:
I suspect that AutomatedLauncherTesting is effectively part of LauncherTestTools. Assuming sig-platform agrees that this code is not ready for use, it may be appropriate to remove this directory as well.

Reply:
This code is definitely not in use.
The LauncherTestTools code is based off of Lumberyard 1.X For example a lot of the paths reference to the "<dev-root>/Bin64vc142" directory, or the `<dev-root>/Cache/<project-name>` directory.

None of these scripts have been updated to work with the different file layout of O3DE.